.github/workflows/unit_test.yaml

```diff
@@ -73,4 +73,4 @@ jobs:
       - name: Run tests
         run: |
           cd test
-          python3 test.py -vvv
+          python3 -m unittest discover unit -vvv
```

Comment:
The output doesn't respect the `-vvv` flag: https://github.com/Ericsson/codechecker_bazel/actions/runs/17065925489/job/48382962683?pr=60